frontend/src/components/LaunchSelect/LaunchSelect.tsx

```diff
@@ -67,13 +69,17 @@ export const LaunchSelect: React.FC<Props> = ({ service, connection }) => {
       <ListItem dense>
         <ListItemIcon></ListItemIcon>
         <Quote margin={0} noInset>
-          <List disablePadding>
+          <List disablePadding className={css.indent}>
             <InlineTemplateSetting
               connection={connection}
               service={service}
               context={app.launchType === 'COMMAND' ? 'copy' : 'launch'}
             />
             <CustomAttributeSettings connection={connection} service={service} />
+            <TestUI>
+              {' '}
```

Comment:
do you need thi empty space?

Reply:
I do not - I think the code formatter adds those sometimes!